### PR TITLE
small narsy fixes

### PR DESCRIPTION
**Pull request by Aranclanos**
Small fixes to narsie, small cleanup of the file
Removed the unused 'contained' var of the singularity

### CODE REVIEW

File: code/modules/power/singularity/narsie.dm [hunk 1]
```diff
@@ -62,16 +59,15 @@
 	godsmack(A)
 	return
 
-/obj/singularity/narsie/proc/godsmack(var/atom/A)
-	if(istype(A,/obj/))
+/obj/singularity/narsie/proc/godsmack(var/atom/A) //you dare stand before a goddess?!
```
Comment (GunHog) [inline code/modules/power/singularity/narsie.dm]: Just go ahead and make it official that Nar-Sie is a female! You have been wanting it for months!

File: code/modules/power/singularity/narsie.dm [hunk 2]
```diff
@@ -62,16 +59,15 @@
...
+	if(isobj(A))
 		var/obj/O = A
 		O.ex_act(1.0)
-		if(O) qdel(O)
-
-	else if(isturf(A))
-		var/turf/T = A
-		T.ChangeTurf(/turf/simulated/floor/plasteel/cult)
-
+		if(O && !qdeleted(O))
```
Comment (Fox-McCloud) [inline code/modules/power/singularity/narsie.dm]: while this is a nifty proc, it's pretty inconsistent with the rest of the codebase; everywhere else explicitly checks for gc_destroyed.

Comment (Aranclanos) [inline code/modules/power/singularity/narsie.dm]: I'll make a new proc and swap all locations where this is checked on a new pr

File: code/modules/power/singularity/narsie.dm [hunk 3]
```diff
@@ -4,12 +4,8 @@
 	icon = 'icons/obj/magic_terror.dmi'
 	pixel_x = -89
 	pixel_y = -85
-	current_size = 9 //It moves/eats like a max-size singulo, aside from range. --NEO
-	contained = 0 //Are we going to move around?
-	dissipate = 0 //Do we lose energy over time?
-	move_self = 1 //Do we move on our own?
-	grav_pull = 5 //How many tiles out do we pull?
```
Comment (phil235) [inline code/modules/power/singularity/narsie.dm]: Why remove this var? it's still used in the eat() proc.

File: code/modules/power/singularity/narsie.dm [hunk 4]
```diff
@@ -131,28 +118,23 @@
 	else
 		target << "<span class ='userdanger'>NAR-SIE HAS CHOSEN YOU TO LEAD HIM TO HIS NEXT MEAL</span>"
 
-//Wizard narsie
+/obj/singularity/narsie/large/attack_ghost(mob/dead/observer/user as mob)
```
Comment (Firecage) [inline code/modules/power/singularity/narsie.dm]: No, bad. BAD Aranclanos.

**Comment by Razharas** [comment]
@Aranclanos 
Fix your pr you lazy

**Comment by Razharas** [comment]
Not a single responce since the day pr was created - ded

**Pull request closed without merging**